config/locales/es.json

```diff
@@ -19,7 +19,14 @@
             "remove": "¡No se está ignorando mas a {user}!"
         },
         "lang": {
-            "title": "Idiomas Disponibles   💬"
+            "success": "Se cambió el lenguaje a `{language}`!  {flag}",
+            "title": "Available languages   💬",
```

Comment:
This is Englando!

Reply:
Just fixed this 🤦‍♂️
Yes it slipped under the radar 🤣